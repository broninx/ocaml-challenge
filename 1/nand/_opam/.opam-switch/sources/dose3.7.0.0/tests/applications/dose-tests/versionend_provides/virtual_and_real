Package: main
Version: 1
Architecture: amd64
Depends: perl-base, libscalar-list-utils-perl (= 1:1.48-1)

Package: perl-base
Version: 5.24.1-5
Architecture: amd64
Provides: libscalar-list-utils-perl (= 1:1.42.02)

Package: libscalar-list-utils-perl
Version: 1:1.48-1
Architecture: amd64

