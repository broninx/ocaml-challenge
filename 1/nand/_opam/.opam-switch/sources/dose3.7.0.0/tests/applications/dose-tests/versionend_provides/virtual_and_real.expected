output-version: 1.2
report:
 -
  package: main
  version: 1
  architecture: amd64
  status: ok
  
 
background-packages: 2
foreground-packages: 1
total-packages: 3
broken-packages: 0
